docs/docs/creating-a-sitemap.md

```diff
@@ -1,8 +1,30 @@
 ---
 title: Creating a sitemap
 ---
+### What is a Site Map?
 
-This is a stub. Help our community expand it.
+An [XML sitemap](https://support.google.com/webmasters/answer/156184?hl=en) lists a website’s important pages, making sure search engines (such as Google) can find and crawl them all, and helping it understand your website structure.
+
+Think of it as a map for your website. It shows what all of the pages are on your website.
+
+### Install
+
+To generate a XML sitemap, We will use the [gatsby-plugin-sitemap](/packages/gatsby-plugin-sitemap/) Package, To install this package, Run the following line on your command line:
+   ```npm install --save gatsby-plugin-sitemap```
```

Comment:
You can highlight console commands like so:

~~~md
```bash
npm install --save gatsby-plugin-sitemap
```
~~~

The formatting is also a bit off here, but that will likely be auto-fixed by Prettier.